website/src/content/docs/docs/getting-started/code-generation-csharp.md

```diff
@@ -0,0 +1,312 @@
+---
+id: code-generation-csharp
+title: Getting Started with TypeSpec Code Generation for C#
```

Comment:
feels a bit generic if we only talk about server code gen. 
It also doesn't seem like it showing up on the website right now and the sidebar title would make it long I think

Reply:
I think we should have code gen guides for clients as well, although probably one that covers all languages rather than individual ones for each language.